freeipa/server/common.sls

```diff
@@ -68,7 +68,7 @@ ldap_logs_access:
       - file: ldap_conf
 {%- endif %}
 
-{%- if not server.ldap.anonymous %}
+{%- if server.ldap.anonymous is not defined %}
```

Comment:
The syntax on this is wrong. If undefined, this throws an error.

Reply:
This is better:
```if not server.ldap.get('anonymous')```